tests/test_execution.py

```diff
@@ -173,8 +176,8 @@ def test_tasks_execution_on_different_organizations(
     assert traintask.status == Status.done
     assert traintask.error_type is None
     assert len(traintask.outputs) == 1
-    output = client_1.get_task_output_asset(traintask.key, OutputIdentifiers.model)
-    assert output.asset is not None
+    with does_not_raise():
```

Comment:
do we need the `does_not_raise` here? 🤔

Reply:
We want to check that `get_task_output_asset` returns an object. This function will either raise an exception or return an object,  so checking if `is not None` seemed less clear to Thibault, as this `output.asset` will never be equals to `None`

Reply:
then, we could just execute the function 🤷 
and add a comment that we just want to check that the function is not raising in this configuration

Reply:
Should I apply this change to all calls to `get_task_output_asset` (previously checking `assert output.asset is not None`), or you meant this one specifically?

Reply:
Should I apply this change to all calls to `get_task_output_asset` (previously checking `assert output.asset is not None`), or you meant this one specifically?

Reply:
Using a context manager to check that a function does not raise seems pretty convoluted for me, so I would change all `with does_not_raise()` 
If we are checking that the correct Exception is raised, it makes sense to keep the context manager.